changelogs/fragments/24.yaml

```diff
@@ -0,0 +1,5 @@
+---
+minor_changes:
+  - Add a ``store_yaml_stream`` function to ``antsibull_core.yaml`` to dump
+    YAML to an IO stream.
```

Comment:
You seem to like these periods before the final brackets :)
```suggestion
    YAML to an IO stream
```

Reply:
It seems I do :)